### PR TITLE
New blog post: "OpenTechSchool at Kiwi PyCon"

### PR DESCRIPTION
**Pull request by projectgus**
Hi everyone!

As discussed here:
https://groups.google.com/a/opentechschool.org/forum/#!topic/discuss.global/nyCVYP0bWnA

Please let me know if it needs any changes.
- Angus

### CODE REVIEW

**Comment by projectgus** [comment]
Does anyone have any thoughts on this? Is it suitable?

I can update the date if need be, as time has moved on a bit now.

Thanks,
- Angus

File: _posts/2013-09-18-kiwi-pycon.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "OpenTechSchool at Kiwi PyCon"
+description: ""
+teaser: "Earlier this month I travlled to Auckland, New Zealand for Kiwi PyCon. I gave a talk entitled **Encouraging New Programmers: The OpenTechSchool Model**, and also organised a free workshop **Introduction to Data Processing with Python**.
```
Comment (lehmannro) [inline _posts/2013-09-18-kiwi-pycon.md]: typo: "I trav **e** lled"

Also, the rest of the teaser is not in sync with the rest of the article (which is not too bad, but might be unintended.)

File: _posts/2013-09-18-kiwi-pycon.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+"
+tags: ["conferences", "outreach"]
+author: angus
+---
+{% include JB/setup %}
+
+Earlier this month I travelled to Auckland, New Zealand for Kiwi PyCon. I spoke about OpenTechSchool and also organised a workshop.
```
Comment (lehmannro) [inline _posts/2013-09-18-kiwi-pycon.md]: Maybe a link to Kiwi PyCon here?

**Comment by lehmannro** [comment]
I've read through the changes and they look good other than for the minor remarks I gave in-line.  Personally, I wouldn't write a blog post in the first person (especially since it's not entirely clear who _I_ is for the OTS blog) but that's just my personal preference.  Feel free to merge.

**Comment by projectgus** [comment]
Eck, should have merged instead of rebased sorry.

Thanks for the comments @lehmannro, have updated the post and will merge now.

**Pull request merged**